test/e2e/federation-ingress.go

```diff
@@ -311,15 +154,15 @@ var _ = framework.KubeDescribe("[Feature:Federation]", func() {
 func equivalentIngress(federationIngress, clusterIngress extensions.Ingress) bool {
 	// TODO: how to decide two ingresses are equal?
 	clusterIngress.Spec = federationIngress.Spec
-	clusterIngress.Spec.ExternalIPs = federationIngress.Spec.ExternalIPs
-	clusterIngress.Spec.DeprecatedPublicIPs = federationIngress.Spec.DeprecatedPublicIPs
-	clusterIngress.Spec.LoadBalancerIP = federationIngress.Spec.LoadBalancerIP
-	clusterIngress.Spec.LoadBalancerSourceRanges = federationIngress.Spec.LoadBalancerSourceRanges
-	// N.B. We cannot iterate over the port objects directly, as their values
-	// only get copied and our updates will get lost.
-	for i := range clusterIngress.Spec.Ports {
-		clusterIngress.Spec.Ports[i].NodePort = federationIngress.Spec.Ports[i].NodePort
-	}
+	//clusterIngress.Spec.ExternalIPs = federationIngress.Spec.ExternalIPs
```

Comment:
Remove dead code.  Also I don't think that this equivalence operator currently works correctly given all of the commented out code?